src/components/ADempiere/PanelDefinition/index.vue

```diff
@@ -42,6 +42,10 @@ export default defineComponent({
       type: String,
       required: true
     },
+    panelMetadata: {
```

Comment:
Greetings @EdwinBetanc0urt The error that was throwing when one opens a window were the following
1. That it was not sending the `parentUuid` in the `props.containerManager.getPanel` method, so the method was returning a undefine and instead it was loading the `panelMetadata`.

2. That in case this condition `if (props.containerManager && props.containerManager.getPanel)` that you put was not fulfilled you were sending to the `props.panelMetadata` so the `panelMetadata` property as it is not registered in the properties the `getPanel()` did not paint the fields and threw an error.

Reply:
Greetings @EdwinBetanc0urt The error that was throwing when one opens a window were the following
1. That it was not sending the `parentUuid` in the `props.containerManager.getPanel` method, so the method was returning a undefine and instead it was loading the `panelMetadata`.

2. That in case this condition `if (props.containerManager && props.containerManager.getPanel)` that you put was not fulfilled you were sending to the `props.panelMetadata` so the `panelMetadata` property as it is not registered in the properties the `getPanel()` did not paint the fields and threw an error.